### DISCUSSION TITLE
Application based APIs Calls Getting 403 Forbidden

### DISCUSSION THREAD

**Discussion by syed-MTalha** [ant-media/Ant-Media-Server#6560] [Q&A] [answered]
I have installed Enterprise AMS on Ubuntu server, and it's up and running. But when it comes to Application calls, it gives me 403. I am trying to use postman and provide JWT Token for Authorization. Please help me how to resolve this issue. I am attaching the screen shot for reference
![Screenshot 2024-08-06 224044](https://github.com/user-attachments/assets/d4624f2c-cfbe-4550-b0ec-8c8a8bdba2e5)
![Screenshot 2024-08-06 224126](https://github.com/user-attachments/assets/cf84fc5e-2e1d-4dc0-95b1-8a8e1c6685ba)

curl --location 'https://studio.taharyuko.com:5443/rest/v2/broadcasts/create?autoStart=false' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json' \
--header 'Authorization: ••••••' \
--header 'Cookie: JSESSIONID=949BDA8EFB8C1F34E93CA648D9F92F4C' \
--data '{
  "streamId": "123456",
  "status": "finished,broadcasting,created",
  "playListStatus": "finished,broadcasting,created",
  "type": "liveStream,ipCamera,streamSource,VoD,playlist",
  "publishType": "WebRTC,RTMP,Pull",
  "name": "<string>",
  "description": "<string>",
  "publish": "<boolean>",
  "date": "<long>",
  "plannedStartDate": "<long>",
  "plannedEndDate": "<long>",
  "duration": "<long>",
  
  
<!doctype html><html lang="en"><head><title>HTTP Status 403 – Forbidden</title><style type="text/css">body {font-family:Tahoma,Arial,sans-serif;} h1, h2, h3, b {color:white;background-color:#525D76;} h1 {font-size:22px;} h2 {font-size:16px;} h3 {font-size:14px;} p {font-size:12px;} a {color:black;} .line {height:1px;background-color:#525D76;border:none;}</style></head><body><h1>HTTP Status 403 – Forbidden</h1><hr class="line" /><p><b>Type</b> Status Report</p><p><b>Message</b> Not authenticated user</p><p><b>Description</b> The server understood the request but refuses to authorize it.</p><hr class="line" /><h3>Apache Tomcat/10.1.19</h3></body></html>

Thanks
Talha

**Comment by amarantmeida** [comment]
@syed-MTalha Please review the guide below for more details on JWT tokens. In the meantime, I will test this on my side and get back to you. 
https://antmedia.io/docs/guides/stream-security/jwt-stream-security-filter/

**Reply by syed-MTalha** [reply]
I have already reviewed the documentation this is for the stream security section. Currently I am trying to get the postman API to work for creation and management of application and streams, and unable to get through that.

**Reply by amarantmeida** [reply]
@syed-MTalha Thank you for your confirmation and I apologise for the delay in getting back to you. Its been a rough couple of days. I will work on this over the weekend and get back to you. In the meantime, do let me know if you need any other assistance.

**Reply by amarantmeida** [reply, marked as answer]
@syed-MTalha Please use the API using the below parameters and headers. Its passing fine for us. 
Note- 
1. Under Body: You can add other parameters as needed. 
2. To generate the JWT token, you can follow the earlier guide which I shared. 

**Request Type**: POST
**URL**: https://test.antmedia.io:5443/test/rest/v2/broadcasts/create
**Headers**:
* Content-Type: application/json
* Authorization: <your JWT token>
**Body (raw JSON)**:
{
   "name": "StreamName"
   "streamId": "123456"
}

Do let me know if this works for you. Will look forward to hearing from you.
